packages/TelegramClient-UI.package/TCUChatWindow.class/instance/addHiddenInfoPage.st

```diff
@@ -0,0 +1,8 @@
+drawing
+addHiddenInfoPage
```

Comment:
I think the name addInfoPage would be fine. Is it really necessary to specify that it is hidden?

Reply:
The reason was, that there were quite a few other add... methods and all of them visually changed the window. In that context, one would expect to also see an info page after calling "addInfoPage" so we wanted to make sure hiding is in the title